bundles/statistics/statsgrid2016/service/ClassificationService.js

```diff
@@ -105,10 +105,7 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.ClassificationService',
             const setBounds = (stats) => {
                 if (opts.method === 'jenks') {
                     // Luonnolliset välit
-                    // geostats.js has performance problems when calculating jenks natural breaks
-                    // hence we are using different function implemented in GeostatsHelper for that
-                    const geostatsHelper = new GeostatsHelper();
-                    response.bounds = geostatsHelper.getJenks(stats.serie, opts.count);
+                    response.bounds = stats.getClassJenks(opts.count);
```

Comment:
Do we need geostatshelper after this at all? Could it be removed?

Reply:
There is no use for GeostatsHelper so it is now removed as obsolete.